ext/hal/nordic/hal/nrf_power.h

```diff
@@ -40,7 +40,7 @@
  * Hardware access layer for (POWER) peripheral.
  */
 #include "nrf.h"
-#include "nordic_common.h"
+//#include "nordic_common.h"
```

Comment:
why? also, commented-out code should not be part of the PR

Reply:
Thanks for catching this, I missed this, I will remove this.